rpc/cloudagent/cloud/etcdcluster/moc_cloudagent_etcdcluster.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
+syntax = "proto3";
+option go_package = "github.com/microsoft/moc/rpc/cloudagent/cloud";
+package moc.cloudagent.etcdcluster;
+
+import "google/protobuf/wrappers.proto";
+import "moc_common_common.proto";
+
+message EtcdClusterRequest {
+	repeated EtcdCluster EtcdClusters = 1;
+	Operation OperationType = 2;
+}
+
+message EtcdClusterResponse {
+	repeated EtcdCluster EtcdClusters = 1;
+	google.protobuf.BoolValue Result = 2;
+	string Error = 3;
+}
+
+message EtcdCluster {
+	string id = 1;
+	string name = 2;
+	string locationName = 3;
```

Comment:
We can also keep it under Group, rather than just location, to provide this as service to clients.
This way each clients can get to create their own etcd clusters like k8s clusters

For your purpose, we can keep it Reserved Group under Reserved Location.

Reply:
Ok, will add group into the parent list.